--- authority
+++ runtime
@@ -1 +1 @@
-<bit-section [formGroup]="sendTextDetailsForm" disableMargin>
+<bit-section disableMargin>
@@ -3,8 +3,2 @@
-    <bit-label>{{ "sendTypeTextToShare" | i18n }}</bit-label>
-    <textarea
-      bitInput
-      id="text"
-      rows="3"
-      formControlName="text"
-      [readonly]="formDisabled()"
-    ></textarea>
+    <bit-label>{{ "i18nTextToShare" | i18n }}</bit-label>
+    <textarea bitInput id="text" rows="3" [value]="value().text" [readonly]="!editing()" (input)="valueChange.emit({ text: inputValue($event) })"></textarea>
@@ -14,2 +8,2 @@
-      <input bitCheckbox type="checkbox" formControlName="hidden" />
-      <bit-label>{{ "hideTextByDefault" | i18n }}</bit-label>
+      <input bitCheckbox type="checkbox" [checked]="value().hidden" [disabled]="!editing()" (change)="valueChange.emit({ hidden: checked($event) })" />
+      <bit-label>{{ "i18nDefaultHideText" | i18n }}</bit-label>
